docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,52 @@
+Read the Docs for Business subscription management
+==================================================
+
+We want to make it easy to manage your billing information.
+You can manage or cancel subscriptions from the :guilabel:`Settings > Subscriptions` page of the :term:`dashboard`.
+
+Managing your subscription
+--------------------------
+
+* Go to https://readthedocs.com/organizations/.
+* Select your organization from this list.
+* Select :guilabel:`Settings` at the top of the page.
+* Select the :guilabel:`Subscription` tab on the left.
+* Click :guilabel:`Manage Subscription`.
+
+This action will take you to the Stripe billing portal where you can manage your subscription.
+
+.. note::
+    You will need to be an organization owner to view subscription information.
+    If you do not have permission,
+    you can ask one of your existing organization owners to make any required changes.
```

Comment:
I've seen this note a couple of places, could consider a new `include::`, but that structure is in `diataxis/main` and can fix it later :+1: